storybookjs/storybook
# fix: Resolve package's types without compiling them first

This is a WIP and doesn't include all packages.

Issue:

## What I did

Added `paths` section to `tsconfig.base.json`. See https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping for more info.

## How to test

- Without running any build commands
- Open https://github.com/storybookjs/storybook/blob/ca2441260bdf2eaf29e6cf58d34bde60856ca7e9/lib/components/src/syntaxhighlighter/syntaxhighlighter.tsx. There should be no TS errors
- Open and remove the following line: https://github.com/storybookjs/storybook/blob/62439ad2a2810876a7fd48e5ed691dbcb79d21a0/lib/theming/src/types.ts#L89
- Now in the `syntaxhighlighter.tsx` should be a TS error visible
